src/css/components/_captions-settings.scss

```diff
@@ -103,3 +100,30 @@
 .vjs-track-settings-controls .vjs-default-button {
   margin-right: 1em;
 }
+
+
+// halfway through vjs-layout-medium side-by-side selects in
+// track-settings-colors collapse, apply fixes as needed
+@media (max-width: 563px) {
```

Comment:
When the selects collapse in `vjs-layout-medium` the right fieldset for font settings is no longer aligned with the left. This is the only fix I could think of, if we don't think it's an issue we can remove it.

Reply:
I'm not sure this really matters. Each column can layout by itself, plus, 4 lines less CSS :)

Reply:
or removing